### ISSUE TITLE
[Accessibility] Only use blockquotes for actual quotes

### ISSUE THREAD

**Issue by frankharkins** [Qiskit/documentation#3653]
## Regular content

We should only use blockquotes for actual quotes, preferably including a citation element (using `<cite>` tags – TBD how best to incorporate into MDX).

Markdown blockquotes (lines starting with `>`) are regularly used for indentation and notes, but they're intended for quotes and our website software currently interprets them as `<blockquote>`. Having non-quoted content in `<blockquote>` tags is wrong and technically an accessibility violation.

We should check we're using all markdown blockquotes correctly.

Here's a list of files with lines starting with `>`, note that some of our code outputs include lines starting with `>`, but this is OK as they're not parsed as blockquotes.

<details><summary>Search results</summary>

```
$ rg '^(\s|")*>\s' -g'{learning/**,docs/*}/*' -l
docs/guides/get-started-with-primitives.ipynb
docs/guides/simulate-with-qiskit-sdk-primitives.ipynb
docs/guides/qpu-information.mdx
docs/guides/introduction-to-qasm.mdx
docs/guides/colibritd-pde.ipynb
docs/guides/primitives-examples.ipynb
docs/guides/bit-ordering.ipynb
docs/tutorials/quantum-approximate-optimization-algorithm.ipynb
docs/migration-guides/qiskit-1.0-installation.mdx
learning/courses/quantum-safe-cryptography/introduction-to-this-course.mdx
learning/courses/quantum-safe-cryptography/asymmetric-key-cryptography.ipynb
learning/courses/quantum-safe-cryptography/quantum-safe-cryptography.ipynb
docs/support/faq.mdx
learning/courses/foundations-of-quantum-error-correction/quantum-code-constructions/other-code-families.ipynb
learning/courses/foundations-of-quantum-error-correction/fault-tolerant-quantum-computing/controlling-error-propagation.ipynb
learning/courses/foundations-of-quantum-error-correction/fault-tolerant-quantum-computing/threshold-theorem.ipynb
learning/courses/foundations-of-quantum-error-correction/quantum-code-constructions/css-codes.ipynb
learning/courses/general-formulation-of-quantum-information/purifications-and-fidelity/purifications.ipynb
learning/courses/general-formulation-of-quantum-information/density-matrices/convex-combinations.ipynb
learning/courses/general-formulation-of-quantum-information/general-measurements/naimark-theorem.ipynb
learning/courses/general-formulation-of-quantum-information/purifications-and-fidelity/fidelity.ipynb
learning/courses/general-formulation-of-quantum-information/quantum-channels/representations-of-channels.ipynb
learning/courses/basics-of-quantum-information/quantum-circuits/limitations-on-quantum-information.ipynb
learning/courses/basics-of-quantum-information/multiple-systems/classical-information.ipynb
learning/courses/fundamentals-of-quantum-algorithms/grover-algorithm/grover-algorithm-description.ipynb
learning/courses/fundamentals-of-quantum-algorithms/quantum-algorithmic-foundations/measuring-computational-cost.ipynb
learning/courses/fundamentals-of-quantum-algorithms/quantum-query-algorithms/deutsch-jozsa-algorithm.ipynb
learning/courses/fundamentals-of-quantum-algorithms/phase-estimation-and-factoring/phase-estimation-problem.ipynb
learning/courses/fundamentals-of-quantum-algorithms/quantum-query-algorithms/deutsch-algorithm.ipynb
learning/courses/fundamentals-of-quantum-algorithms/quantum-algorithmic-foundations/factoring-and-gcd.ipynb
learning/courses/fundamentals-of-quantum-algorithms/quantum-query-algorithms/simon-algorithm.ipynb
learning/courses/fundamentals-of-quantum-algorithms/grover-algorithm/unstructured-search.ipynb
learning/courses/fundamentals-of-quantum-algorithms/phase-estimation-and-factoring/shor-algorithm.ipynb
```

</details>

## API docs

There are lots of cases of this in the API docs, most seem unintentional (they're not marked as quotes in the original source).

For example, there's a blockquote in the `broadcast_arguments` method:

* [Website](https://quantum.cloud.ibm.com/docs/en/api/qiskit/qiskit.circuit.library.XGate#broadcast_arguments)
* [Markdown](https://github.com/Qiskit/documentation/blob/61e55b53a18c9637e69b4177137a40ce57151f9d/docs/api/qiskit/qiskit.circuit.library.XGate.mdx?plain=1#L154-L172)
* [RST source](https://github.com/Qiskit/qiskit/blob/1e49fbdd265e7cac36263ce21eef6cabd288bafa/qiskit/circuit/gate.py#L200-L212)

I guess Sphinx interpets the indentation in the RST source as a quote.

**Comment by beckykd** [comment]
□	docs/guides/get-started-with-primitives
□	docs/guides/simulate-with-qiskit-sdk-primitives
□	docs/guides/qpu-information
□	docs/guides/introduction-to-qasm
□	docs/guides/colibritd-pde
□	docs/guides/primitives-examples
□	docs/guides/bit-ordering
□	docs/tutorials/quantum-approximate-optimization-algorithm
□	docs/migration-guides/qiskit-1.0-installation
□	learning/courses/quantum-safe-cryptography/introduction-to-this-course
□	learning/courses/quantum-safe-cryptography/asymmetric-key-cryptography
□	learning/courses/quantum-safe-cryptography/quantum-safe-cryptography
□	docs/support/faq
□	learning/courses/foundations-of-quantum-error-correction/quantum-code-constructions/other-code-families
□	learning/courses/foundations-of-quantum-error-correction/fault-tolerant-quantum-computing/controlling-error-propagation
□	learning/courses/foundations-of-quantum-error-correction/fault-tolerant-quantum-computing/threshold-theorem
□	learning/courses/foundations-of-quantum-error-correction/quantum-code-constructions/css-codes
□	learning/courses/general-formulation-of-quantum-information/purifications-and-fidelity/purifications
□	learning/courses/general-formulation-of-quantum-information/density-matrices/convex-combinations
□	learning/courses/general-formulation-of-quantum-information/general-measurements/naimark-theorem
□	learning/courses/general-formulation-of-quantum-information/purifications-and-fidelity/fidelity
□	learning/courses/general-formulation-of-quantum-information/quantum-channels/representations-of-channels
□	learning/courses/basics-of-quantum-information/quantum-circuits/limitations-on-quantum-information
□	learning/courses/basics-of-quantum-information/multiple-systems/classical-information
□	learning/courses/fundamentals-of-quantum-algorithms/grover-algorithm/grover-algorithm-description
□	learning/courses/fundamentals-of-quantum-algorithms/quantum-algorithmic-foundations/measuring-computational-cost
□	learning/courses/fundamentals-of-quantum-algorithms/quantum-query-algorithms/deutsch-jozsa-algorithm
□	learning/courses/fundamentals-of-quantum-algorithms/phase-estimation-and-factoring/phase-estimation-problem
□	learning/courses/fundamentals-of-quantum-algorithms/quantum-query-algorithms/deutsch-algorithm
□	learning/courses/fundamentals-of-quantum-algorithms/quantum-algorithmic-foundations/factoring-and-gcd
□	learning/courses/fundamentals-of-quantum-algorithms/quantum-query-algorithms/simon-algorithm
□	learning/courses/fundamentals-of-quantum-algorithms/grover-algorithm/unstructured-search
□	learning/courses/fundamentals-of-quantum-algorithms/phase-estimation-and-factoring/shor-algorithm
